app/routes-config.js

```diff
@@ -15,10 +15,17 @@ export const user_answers = id => `/users/${id}#answers`;
 export const user_settings = id => `/users/${id}/settings`;
```

Comment:
I missed that part earlier. The names with `_` are inconsistent with the rest of the code. Also `uniqueAnswerId` is named differently.